src/main/java/net/fabricmc/example/mixin/ExampleMixin.java

```diff
@@ -1,15 +1,25 @@
 package net.fabricmc.example.mixin;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.TitleScreen;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(TitleScreen.class)
-public class ExampleMixin {
-	@Inject(at = @At("HEAD"), method = "init()V")
+public abstract class ExampleMixin extends Screen {
+	private MixinTitleScreen(Text title) {
```

Comment:
Did you try compiling this? The constructor doesn't have the same name as the class.

Reply:
nlg, no. I saw the adjustments on the Wiki and came straight here - as in edited on GH. V glad that people actually spot the little things.